client/status.go

```diff
@@ -0,0 +1,76 @@
+package client
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/google/subcommands"
+)
+
+const (
+	// ExitSuccess represents no error.
+	ExitSuccess subcommands.ExitStatus = 0
```

Comment:
why not reference directly `subcommands.ExitSuccess`, `subcommands.ExitFailure`, and `subcommands.ExitUsageError`?

```go
const (
    ExitSuccess subcommands.ExitStatus = subcommands.ExitSuccess
    ExitFailure = subcommands.ExitFailure
    ExitUsageError = subcommands.ExitUsageError
    ExitInvalidParams = 3
    ...
```